### ISSUE TITLE
Teams UI

### ISSUE THREAD

**Issue by LanesGood** [developmentseed/osm-teams#51]
@sethvincent @kamicut I've uploaded my edits to the OSM Teams UI to Invision here:
https://invis.io/3AT7GFB5XJQ#/376319453_ExplorePage

I'll post screenshots below too, but like making use of the InVision version control/updating features for design. 

There are still some kinks to work through on the navigation, and as fleshed-out wireframes these screens still need color applied. Overall I think this is a good middle ground between the original developer-centric style and a material design look. The Teams list page is demonstrating a few different possible sidebar states, where the filters would be housed. I don't yet know what will go in the individual team page sidebar.

The circles present next to teams, members and in the top header bar would be replaced with avatar images, while the circles in the left sidebar would be replaced with customized icons representing the navigation actions. The Teams logo is currently a placeholder.

![ExplorePage](https://user-images.githubusercontent.com/12634024/62076704-536a2280-b216-11e9-9949-b231491302e7.png)
![ExplorePage w Sidebar](https://user-images.githubusercontent.com/12634024/62076703-536a2280-b216-11e9-8058-442fd95ad68f.png)
![ExplorePage w Filters](https://user-images.githubusercontent.com/12634024/62076702-536a2280-b216-11e9-9436-d0807c8f4588.png)
![Team Page - Admin](https://user-images.githubusercontent.com/12634024/62076705-536a2280-b216-11e9-9162-75c49babb778.png)

**Comment by kamicut** [comment]
Would the active campaigns be manually added? I don't imagine we can support pulling from all tasking managers, or figuring out which team is mapping what campaign automatically.

**Comment by LanesGood** [comment]
@kamicut this is just a potential content area - Perhaps this could instead be a photos or events section?

**Comment by LanesGood** [comment]
@kamicut @sethvincent I've updated the Invision link with the sidebar navigation in collapsed, and expanded states, and "My teams" list with action  states included. Let me know if you have any thoughts on this suggested interaction and user selection behavior.

I've also added in some initial styles for the Explore and Team screens. These are pretty boxy by design, but we can lighten this if you think it distracts from the intended interactivity of the dashboard.

https://invis.io/3AT7GFB5XJQ#/377047076_Sidebar_Collapsed
